### ISSUE TITLE
What the bot expect to do?

### ISSUE THREAD

**Issue by rever96** [sudoguy/tiktok_bot#5]

**Comment by reliefs** [comment]
if i can login i will start to implement these functions.. :) It's really a must

**Comment by rever96** [comment]
will u make the further improvements of this repo public? or at least could u give me some access to your improvements?

**Comment by Abdulsamipy** [comment]
hi, can you guys tell me how i can install this bot, extracted the file via pip but what to do next .. i've tried to run bot.py but no response can you guide me .. thanks

**Comment by reliefs** [comment]
yes if i make any functions or code to this repo i will push them :)
